Demo/src/main/java/com/braintreepayments/demo/PayPalRequestFactory.java

```diff
@@ -39,6 +41,11 @@ public static PayPalVaultRequest createPayPalVaultRequest(Context context) {
             request.setShippingAddressOverride(postalAddress);
         }
 
+        PayPalNativeCheckoutConfig nativeConfig = new PayPalNativeCheckoutConfig();
+        nativeConfig.setCorrelationId(UUID.randomUUID().toString());
+        nativeConfig.setReturnUrl("com.braintreepayments.demo://paypalpay");
+
+        request.setNativeConfig(nativeConfig);
```

Comment:
Can we encapsulate this logic within `PayPalNativeRequest` and add a `returnUrl` property to simplify the interface?

Reply:
yup, done.